docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,366 @@
+---
+sidebar_label: "Required Permissions"
+title: "Required Permissions"
+description: "Review the required permissions for deploying clusters on Azure"
+icon: ""
+hide_table_of_contents: false
+tags: ["public cloud", "azure", "permissions"]
+sidebar_position: 100
+---
+
+
+Palette requires a set of permissions to properly deploy and manage the lifecycle of clusters deployed on Azure. We recommend you create a custom role with all the required permissions for Palette or VerteX to deploy and manage clusters on Azure. Create a [role assignment](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments) that has the service principal you want to use with Palette and the appropriate [scope level](https://learn.microsoft.com/en-us/azure/role-based-access-control/scope-overview). By using the subscription as the scope level for the role assignment, you can ensure all use cases for Palette and VerteX are supported. 
+
+:::warning
+
+A custom role allows you to limit the permissions to only what is required by Palette. If you don't want to create a custom role, you can assign the Azure  build-in [Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role to the service principal used by Palette. The Contributor role has all the required permissions, but it also has additional permissions that are not required by Palette. 
+
+:::
+
+
+The following use cases are supported:
+
+- [Static Placement](#static-placement) - Palette will deploy clusters on a pre-existing network resources specified by you.
+
+- [Dynamic Placement](#dynamic-placement) - Palette will create the network resources required for the cluster.
+
+- [Validator](#validator) - Use the Validator open-source framework to validate the permissions required by Palette.
+
+## Static Placement
+
+Static placement is when you have a pre-existing network resource group, VNet, subnets and security groups you want Palette to use when deploying a cluster. Static placement supports three use cases. The required actions are all the same but the scope levels for each use case are different.
```

Comment:
```suggestion
Choose static placement when you want Palette to use pre-existing network resource groups, VNets, subnets, and security groups.
```

Reply:
Comments on the structure.

Given that we offer three options that seem to be at the same level, it makes sense to standardize the information we provide. Here's what I would do:

- Name the option (e.g., Static placement).
- Offer a broad use case (e.g., when you want Palette to use pre-existing network resource groups, VNets, subnets, and security groups).
- Introduce sub-options, if any.
- List the required actions and their scopes.

And a small note: usually, counting things in docs is not helpful unless there's an explicit need for that. When you say, "Palette supports three use cases," that doesn't add much for our readers. It can also be difficult to support in the future—when the number of use cases changes, it can be hard to find and update these sentences. Instead, we should name the use cases and treat them as a lead-in to whatever content follows.